### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by anupambhatnagar**
Adding links to HTA to readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,33 @@
...
+A central component is Libkineto, a profiling library with special focus on low-overhead GPU timeline tracing.
 
 ## Libkineto
+
 Libkineto is an in-process profiling library integrated with the PyTorch Profiler. Please refer to the [README](libkineto/README.md) file in the `libkineto` folder as well as documentation on the [new PyTorch Profiler API](https://pytorch.org/docs/master/profiler.html).
 
-## PyTorch TensorBoard Profiler
+## Holistic Trace Analysis
+
+Holistic Trace Analysis(HTA) is an open source project performance debugging library aimed at
```
Comment (aaronenyeshi) [inline README.md:19]: Small nit: add spacing before bracket and remove project
> Holistic Trace Analysis (HTA) is an open source performance debugging library aimed at

File: README.md [hunk 2]
```diff
@@ -2,20 +2,33 @@
...
+distributed workloads. HTA takes as input PyTorch Profiler traces and elevates the performance
+bottlnecks to enable faster debugging. Here's a partial list of features in HTA:
```
Comment (aaronenyeshi) [inline README.md:21]: typo: bottlenecks

**Review by aaronenyeshi — CHANGES_REQUESTED** [review]
Overall looks great, see the small nits.

File: README.md (hunk 2 above)
Comment (anupambhatnagar) [reply README.md]: comments addressed.

**Review by aaronenyeshi — APPROVED** [review]
LGTM!

**Pull request closed without merging**